Source/WebCore/dom/Traversal.h

```diff
@@ -36,6 +36,7 @@ class NodeIteratorBase {
 public:
     Node& root() { return m_root.get(); }
     const Node& root() const { return m_root.get(); }
+    Ref<Node> protectedRoot() const { return m_root.copyRef(); }
```

Comment:
We don't need the `.copyRef()` I believe.

Reply:
Ditto.